client/src/components/common/helpers/index.js

```diff
@@ -0,0 +1,13 @@
+import { Form, TextArea, Button, FormField, Dropdown, Checkbox, Label } from 'semantic-ui-react';
+import TextAreaAutoResize from 'react-textarea-autosize';
+
+export {
+  Form,
+  TextArea,
+  Button,
+  TextAreaAutoResize,
+  FormField,
+  Dropdown,
+  Checkbox,
+  Label,
+};
```

Comment:
Abstracting components from a library and keeping it in one place is good but then we can directly import from here, instead of creating another component which wrap these component and simply passing props. @nimish-gupta views?

Reply:
Yeah you can just import here all the components and then pass it. You can skip wrapping components if you are not modifying the props